src/ActiveQuery.php

```diff
@@ -107,6 +107,15 @@ class ActiveQuery extends Query implements ActiveQueryInterface
     private array $joinWith = [];
     private ActiveRecordInterface|null $arInstance = null;
 
+    /**
+     * @param class-string $arClass
```

Comment:
```suggestion
     * @param string $arClass
```

Psalm annotations should be specified with `psalm-` prefix, e.g. `@psalm-param`. For `$arClass` it is already exists.
PHPDoc type annotations specified without `psalm-` prefix should be correct, e.g. `@param string`